bika/lims/browser/js/bika.lims.loader.js

```diff
@@ -89,6 +89,10 @@ window.bika.lims.controllers =  {
     ".template-bika-lims-content-samplingsround.portaltype-client":
         ['ClientSamplingRoundAddEditView'],
 
+    // Client Samples View
+    ".portaltype-client.template-samples":
+        ['ClientSamplesView'],
```

Comment:
This one is to handle the print button on top, right? Why don't you bind that event in `bika.lims.samples.js` to `print_samples_trigger`, because the code is the same

Reply:
@ramonski I kept it in both places so that the structure of the functionality (in terms of code) is coherent, if that can be said. When inspecting the code I would find it both strange and harder to follow if the js code from the Samples listing view was binded to a function located in the client js code.

I would say that the logical way of structuring the code, which results in it being easier to understand and follow is: 
`js` code for the Clients listing view -> `bika.lims.client.js`
`js` code for the Samples listing view -> `bika.lims.samples.js`

Maybe the function could be moved to `bika.lims.common.js` or a similar file and then be used in both situations?

Reply:
@ramonski I finally did as you suggested